keras2onnx/parser.py

```diff
@@ -20,7 +20,7 @@
 def _infer_variable_type(tensor):
     tensor_shape = []
     if tensor.shape not in (tf.TensorShape(None), tf.TensorShape([])):
-        tensor_shape = [d.value for d in tensor.shape]
+        tensor_shape = [d.value if d.value is not None else 'None' for d in tensor.shape]
```

Comment:
This will break the right behavior was designed in ONNX. If the right way cannot work for WINML, we need add a program option to be backward compatible with WINML.